grails-app/controllers/ygor/EnrichmentController.groovy

```diff
@@ -72,6 +72,7 @@ class EnrichmentController {
         def foDelimiter = request.parameterMap['formatDelimiter'][0]
         def foQuote     = null // = request.parameterMap['formatQuote'][0]
         def foQuoteMode = null // = request.parameterMap['formatQuoteMode'][0]
+        def dataTyp     = request.parameterMap['dataTyp'][0]
```

Comment:
Generally, we want to word variables and code in English. So it would be `dataType`. But OK for me to leave it for now like it is.